fhir-persistence-schema/src/main/resources/postgres/add_any_resource.sql

```diff
@@ -57,8 +57,8 @@ BEGIN
     -- remember that we have a concurrent system...so there is a possibility
     -- that another thread snuck in before us and created the logical resource. This
     -- is easy to handle, just turn around and read it
-    INSERT INTO {{SCHEMA_NAME}}.logical_resources (logical_resource_id, resource_type_id, logical_id)
-         VALUES (v_logical_resource_id, v_resource_type_id, p_logical_id) ON CONFLICT DO NOTHING;
+    INSERT INTO {{SCHEMA_NAME}}.logical_resources (logical_resource_id, resource_type_id, logical_id, reindex_tstamp)
+         VALUES (v_logical_resource_id, v_resource_type_id, p_logical_id, '1970-01-01') ON CONFLICT DO NOTHING;
```

Comment:
should we add it with the current/"now" timestamp instead?

Reply:
Actually I wanted to reflect the fact that the resource had never been "reindexed". Initially I had tried to use NULL, but that complicated the execution plan of the query I was using to identify which resource to process next and caused it to break when we got near the end of processing. This epoch date is therefore the best way to handle this.

Reply:
Got it

Reply:
> Actually I wanted to reflect the fact that the resource had never been "reindexed".

What is the benefit of knowing that?  Would it be more important to know "at which point in time did we extract the search parameters" (so that humans can correlate that with the config at the time)?

The consequence of this decision is that we'll reindex everything at least once. consider the following example:
1. user changes the search config at tstamp 2020-09-04
2. user continues to load data
3. user says "please reindex everything that was written before 2020-09-04"
4. system reindexes everything, regardless of when it was written

I'm ok with this decision as long as we are ok with that consequence

Reply:
We can make it more sophisticated in the future if we need to handle other scenarios. My guess was that you'd pretty much be reindexing everything anyway.